src/MultiRegion/multi_region_grid.jl

```diff
@@ -33,6 +34,18 @@ const MultiRegionGrids = Union{MultiRegionGrid, ImmersedMultiRegionGrid}
 # Convenience
 @inline Base.getindex(mrg::MultiRegionGrids, r)  = getregion(mrg, r)
 @inline Base.first(mrg::MultiRegionGrids)  = mrg[1]
+@inline Base.lastindex(mrg::MultiRegionGrids)  = length(mrg)
+
+function minimum_spacing(dir, grid::MultiRegionGrid, ℓx, ℓy, ℓz)
+    LX, LY, LZ = map(destantiate, (ℓx, ℓy, ℓz))
+
+    min_Δ = Inf
+    for region in 1:lastindex(grid)
+        min_Δ = min(min_Δ, minimum_spacing(dir, grid[region], ℓx, ℓy, ℓz))
+    end
```

Comment:
```suggestion
    min_Δ = minimum(minimum_spacing(dir, grid[r], ℓx, ℓy, ℓz) for r=1:lastindex(grid))
```

PS implement some function that returns the number of regions? Eg `regions`, `panels`, `number_of_regions`?

Reply:
length(grid) works but julia was telling me “we suggest not using length in a for loop, use lastindex”